public/javascripts/privateSongForm.js

```diff
@@ -0,0 +1,219 @@
+                            onclick: function() {
+                                var isConfirmed = true;
+                                $('#stanzas').children().each(function() {
+                                    if ($(this).find('textarea').val().replace(/\s/g, '') !== '') {
+                                        isConfirmed = confirm('Do you want to overwrite your existing lyric ?')
+                                        return;
+                                    }
+                                })
+                                if (isConfirmed) {
+                                    lyric = copypaste
+                                    $('#copypaste').modal('hide')
+                                    $('#copypaste').find('textarea').val('')
+                                }
+                            }
+                        }, 'Done')
+                    ])
+                ])
+            ])
+        ])
+    }
+
+    function displayError() {
+        if (!_.isEmpty(errors)) {
+            return m('#alert.alert.alert-danger', [
+                errors.map((error) => {
+                    return m('p', error)
+                })
+            ])
+        }
+    }
+
+    var privateSongFormComponent = {
+        view: function() {
+            return [
+                displayError(),
+                m('div', {
+                    style: {
+                        'font-size': '15px'
+                    }
+                }, [
+                    m('.form-group', [
+                        m('label', 'Title'),
+                        m('input.form-control', {
+                            value: title(),
+                            readonly: readonly,
+                            onchange: m.withAttr('value', title),
+                            config: function(elem, isInit) {
+                                if (!isInit) {
+                                    if (readonly) {
+                                        $(elem).prop('readonly', true)
+                                    }
+                                }
+                            }
+                        })
+                    ]),
+                    m('.form-group', [
+                        m('label', 'Author'),
+                        m('input.form-control', {
+                            value: author(),
+                            readonly: readonly,
+                            onchange: m.withAttr('value', author),
+                            config: function(elem, isInit) {
+                                if (!isInit) {
+                                    if (readonly) {
+                                        $(elem).prop('readonly', true)
+                                    }
+                                }
+                            }
+                        })
+                    ]),
+                    m('.form-group', [
+                        m('label', 'Language'),
```

Comment:
I imagine this component might be reusable in other places in the app. Consider making it a reusable component.